front/app/containers/Admin/communityMonitor/components/LiveMonitor/components/FormResults.tsx

```diff
@@ -1,64 +1,103 @@
 import React from 'react';
 
-import { Box, Text } from '@citizenlab/cl2-component-library';
-import { useParams } from 'react-router-dom';
+import { Box, Icon, Text } from '@citizenlab/cl2-component-library';
+import { useParams, useSearchParams } from 'react-router-dom';
 
 import useProjectById from 'api/projects/useProjectById';
 import useFormResults from 'api/survey_results/useSurveyResults';
 
 import SentimentQuestion from 'components/admin/FormResults/FormResultsQuestion/SentimentQuestion';
 
-import { useIntl } from 'utils/cl-intl';
-
-import messages from '../messages';
+import {
+  categoryColors,
+  getQuarterFilter,
+  getYearFilter,
+} from './HealthScoreWidget/utils';
 
 type Props = {
   projectId?: string;
   phaseId?: string;
 };
 
 const FormResults = (props: Props) => {
+  // Get the projectId and phaseId from the URL
   const { projectId: projectIdParam, phaseId: phaseIdParam } = useParams() as {
     projectId: string;
     phaseId: string;
   };
-
   const projectId = props.projectId || projectIdParam;
   const phaseId = props.phaseId || phaseIdParam;
 
-  const { formatMessage } = useIntl();
   const { data: project } = useProjectById(projectId);
 
+  // Get the current year and quarter for the results
+  const [search] = useSearchParams();
+
+  // Get the year and quarter
+  const year = getYearFilter(search);
+  const quarter = getQuarterFilter(search);
+
+  // Fetch the form results
   const { data: formResults } = useFormResults({
     phaseId,
     filterLogicIds: [],
+    quarter: parseInt(quarter, 10),
+    year: parseInt(year, 10),
   });
 
   if (!formResults || !project) {
     return null;
   }
 
-  const { totalSubmissions, results } = formResults.data.attributes;
+  const { results } = formResults.data.attributes;
 
+  // Filter the results to only include sentiment questions
   const sentimentQuestionResults = results.filter(
     (question) => question.inputType === 'sentiment_linear_scale'
   );
 
+  const isFirstQuestionInCategory = (category: string, index: number) => {
+    return (
+      sentimentQuestionResults.at(index - 1)?.questionCategory !== category
+    );
+  };
+
```

Comment:
This will change once the BE returns the Categories as pages, but I'll clean this up once the BE is implemented :)